packages/@react-aria/interactions/src/textSelection.ts

```diff
@@ -67,7 +67,7 @@ export function restoreTextSelection(target?: Element) {
       // for the whole page in the middle of the animation and cause jank.
       runAfterTransition(() => {
         // Avoid race conditions
-        if (state === 'restoring') {
+        if (state === 'restoring' && target) {
```

Comment:
I'm worried that this is a pitfall in our new iframe support, if no target is passed in, how do we know which document to act on. I'd have liked to instead of checking for target here, changed the next line to
```
const documentObject = target ?  getOwnerDocument(target) : document;
```
however, if it's in an iframe, I am not sure how we know that in which case we'd leak the style and possibly incorrectly change it on whatever the root document is.

Reply:
@slye-stripe have y'all come across this or already fixed it in your patches?

Reply:
oo, it broke a test too
```
FAIL packages/@react-aria/interactions/****/useMove.****.js (11.288 s)
  ● useMove › user-select: none › adds and removes user-select: none to the body (iOS)

    expect(received).toBe(expected) // Object.is equality

    Expected: "none"
    Received: "contain"

      213 |       expect(document.documentElement.style.webkitUserSelect).toBe(mockUserSelect);
      214 |       fireEvent.touchStart(el, {changedTouches: [{identifier: 1, pageX: 1, pageY: 30}]});
    > 215 |       expect(document.documentElement.style.webkitUserSelect).toBe('none');
          |                                                               ^
      216 |       fireEvent.touchMove(el, {changedTouches: [{identifier: 1, pageX: 10, pageY: 25}]});
      217 |       expect(document.documentElement.style.webkitUserSelect).toBe('none');
      218 |       fireEvent.touchEnd(el, {changedTouches: [{identifier: 1, pageX: 10, pageY: 25}]});

      at Object.toBe (packages/@react-aria/interactions/****/useMove.****.js:215:63)
```

I've pushed the change I made earlier, but I think it doesn't solve the actual problem

Reply:
Not that I know of yet. Perhaps if users are portaling into an iframe, we could encourage them to pass in a target, which wouldn't break existing tests! 

Also, the `target instanceof HTMLElement || target instanceof SVGElement` is interesting because the HTMLElement prototype could be different from one window object to another so we've done `target instanceof windowObject.HTMLElement || target instanceof windowObject.SVGElement` in our patches as well

Reply:
yeah, that'd be the issue. It's not the most used feature probably, so hard to remember to test for it, even manually.

I did realize after logging off that your getOwnerDocument already does the fallback, so this is purely a typescript issue in terms of the changes I need to make in this PR.

The issue I pointed out still exists, and it'd be good to know if it causes a problem, but I'm no longer as worried about it after a good night's sleep.